x/axelarnet/types/msg_call_contract.go

```diff
@@ -44,6 +45,17 @@ func (m CallContractRequest) ValidateBasic() error {
 		return fmt.Errorf("contract address empty")
 	}
 
+	if m.Fee != nil {
+
+		if err := sdk.VerifyAddressFormat(m.Fee.Recipient); err != nil {
+			return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, sdkerrors.Wrap(err, "fee recipient").Error())
+		}
+
+		if !m.Fee.Amount.IsValid() {
```

Comment:
Check if it's positive instead

Reply:
`IsValid()` does some check on the denom though too, we should keep that though right?